pkg/gui/controllers/files_controller.go

```diff
@@ -90,6 +90,11 @@ func (self *FilesController) GetKeybindings(opts types.KeybindingsOpts) []*types
 			Handler:     self.checkSelectedFileNode(self.ignore),
 			Description: self.c.Tr.LcIgnoreFile,
 		},
+		{
+			Key:         opts.GetKey(opts.Config.Files.ExcludeFile),
+			Handler:     self.checkSelectedFileNode(self.exclude),
+			Description: self.c.Tr.LcIgnoreFile,
```

Comment:
Shouldn't this be `ExcludeFile` translation? Maybe that:s the reason why you can't find it in the menu :)

Reply:
@mark2185 🤦 how did I miss that? I should stop pushing code and go to bed. It's almost midnight here. The good news is I  don't have to push any more code because its now working so I'm going to take this PR out of draft so it can be reviewed :)